lib/AACStream/aacs.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * mux.js
+ *
+ * Copyright (c) 2016 Brightcove
+ * All rights reserved.
+ *
+ * A stream-based aac to mp4 converter. This utility can be used to
+ * deliver mp4s to a SourceBuffer on platforms that support native
+ * Media Source Extensions.
+ */
+'use strict';
+var Stream = require('../utils/stream.js');
+
+var
+
+/**
+ * Splits an incoming stream of binary data into ADTS and ID3 Frames.
+ */
+
+ AacStream = function() {
+  var
+    everything,
+    receivedTimeStamp = false,
+    timeStamp = 0;
+
+  AacStream.prototype.init.call(this);
+
+  this.setTimestamp = function (frame) {
```

Comment:
Calling this method setTimestamp() implies some behaviors that wouldn't be true in this case. It's a common convention to create "setters" that exist to allow downstream developers to mutate some internal properties of an object. In that case though, the setter would normally take as an argument the value that is being set. I think it's a bit confusing to call this setTimestamp() and have it take a frame, given that practice.

How about splitting the side-effect portion of this out from the calculate-timestamp-from-frame portion? Something like:
```js
this.calculateTimestamp_ = function(frame) {
  /* ... */ 
};

// later, when you need to set the timestamp:
timestamp = this.calculateTimestamp_(frame);
```